test/functional/setup_http_server

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
```

Comment:
A grammar nit: "setup" is a noun, "set up" is a verb. Thus, this should be named set_up_http_server. :-)